### ISSUE TITLE
Do you plan to do Canaries?

### ISSUE THREAD

**Issue by sielaq** [CiscoCloud/mesos-consul#8]
Hi,

we have project https://github.com/eBayClassifiedsGroup/PanteraS (PaaS in a box)
where we use registrator, but since with registartor is hard to get any feature :)
we would like to switch for something more maintainable.

We were able to do canaries due to possibility to register with different name
(SERVICE_NAME in ENV in marathon deployment plan).

Do you plan to do something similar, using tags or env, so consul service name is different than marathon id ?

**Comment by stevendborrelli** [comment]
@sielaq we'd like to implement that feature. 
The main problem is that currently, the mesos state.json file does not show environment variables (also port mapping does not work very well). 

We're considering using mesos labels for the same feature.

**Comment by sielaq** [comment]
Your project looks cool too :) sound like we make PaaS using similar stack,
which prove that stack we both have chosen a proper stack .  
... and we started same time even :) ...

We tried to simplify it as much as possible -> we have one container
https://registry.hub.docker.com/u/panteras/paas-in-a-box/
so developers can use the same container on their laptops,
Your project seems to be more complex - "big clusters" oriented.

My colleague work on consul router in golang now,
so consul-template and HAproxy will be replaced.
I hope he will OpenSource it soon.
We also gonna include vault soon.

I'm sure we can collaborate and share each other.
